subprojects/build-init/src/main/java/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -141,8 +141,8 @@ private void writeWrapperTo(File destination) {
         if (jarFileSource == null) {
             throw new GradleException("Cannot locate wrapper JAR resource.");
         }
-        try (InputStream stream = jarFileSource.openStream()) {
-            Files.copy(stream, destination.toPath(), REPLACE_EXISTING);
+        try (InputStream in = jarFileSource.openStream(); OutputStream out = new FileOutputStream(destination)) {
```

Comment:
I guess `Files.newOutputStream` has the same problem as `Files.copy`? Or can the problem be fixed by providing the right options to `Files.newOutputStream`?

Do you understand the differences between `Files` and when using `FileOutputStream`?